### PR TITLE
logplot: write down average running rate to avg-running-rate.txt

### PR DESCRIPTION
**Pull request by hongchaodeng**
Manually test it in local environment.
It will write results to a file with 6 digit precision like:

```
13.827412
```

I'm expecting to use this to pass around digital results and then compare daily against regression.

### CODE REVIEW

**Comment by hongchaodeng** [comment]
@xiang90

File: logplot/main.go [hunk 1]
```diff
@@ -174,6 +176,23 @@ func plotRunningRateVsPods(rs []densityResult) {
...
+	}
+
+	r := getAvgRunningRate(rs)
+	s := strconv.FormatFloat(r, 'f', 6, 64)
+	_, err = f.WriteString(s)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "ERROR: failed to write to 'avg-running-rate.txt'")
+		os.Exit(1)
+	}
+	fmt.Println("successfully write average running rate to avg-running-rate.txt")
```
Comment (xiang90) [inline logplot/main.go:196]: sync and close f after writing.

File: logplot/main.go [hunk 2]
```diff
@@ -174,6 +176,25 @@ func plotRunningRateVsPods(rs []densityResult) {
...
 }
 
+func recordAvgRunningRate(rs []densityResult) {
+	f, err := os.Create("avg-running-rate.txt")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "ERROR: cannot create 'avg-running-rate.txt' file")
+		os.Exit(1)
+	}
+
+	defer f.Close()
```
Comment (hongchaodeng) [inline logplot/main.go:187]: @xiang90 
Addressed here.

File: logplot/main.go [hunk 3]
```diff
@@ -14,6 +15,8 @@ import (
 	"github.com/gonum/plot/vg"
 )
 
+var totalPods int
```
Comment (xiang90) [inline logplot/main.go:19]: not a fan of global, but seems to be just fine... At lease we should doc it.

File: logplot/main.go [hunk 4]
```diff
@@ -241,3 +262,13 @@ func getRunningRatePoints(rs []densityResult) plotter.XYs {
 	}
 	return pts
 }
+
+func getAvgRunningRate(rs []densityResult) float64 {
+	for i := range rs {
+		if rs[i].running >= totalPods {
+			return float64(rs[i].running) / float64((i+1)*10)
```
Comment (xiang90) [inline logplot/main.go]: let's define 10 as reporting Interval or something like that.

**Comment by xiang90** [comment]
LGTM asides from nits.

**Comment by hongchaodeng** [comment]
Addressed all comments.

**Comment by xiang90** [comment]
LGTM

**Pull request merged**